searx/static/themes/simple/src/js/main/searx_keyboard.js

```diff
@@ -1,12 +1,12 @@
-/*global searx*/
```

Comment:
Nit: all files we touch should become a SPDX-License-Identifier, if there is no Copyright a simple tag should be enough:

    /* SPDX-License-Identifier: AGPL-3.0-or-later */

Reply:
updated